scripts/update-abi-registry.js

```diff
@@ -17,6 +17,12 @@ async function fetchNodeVersions () {
   const schedule = await getJSONFromCDN('nodejs/Release/schedule.json')
   const versions = {}
 
+  // In Node 17.0.0, a header is missing and causes an error during compilation.
+  // See https://github.com/electron/node-abi/issues/114
+  const targetPinning = {
+    '17.0.0': '17.0.1'
```

Comment:
suggestion: put the specific comment next to the override. There should be a comment here describing what the mapping does in general and how it's supposed to work (but I didn't add that in the suggestion below).

Also, I prefer calling it `targetOverride` over `targetPinning`.

```suggestion
  const targetOverride = {
    // In Node 17.0.0, a header is missing and causes an error during compilation.
    // See https://github.com/electron/node-abi/issues/114
    '17.0.0': '17.0.1'
```